### PR TITLE
Suggest fix

### PR DESCRIPTION
**Pull request by LucGenetier**
Takes advantage of the Culture to sort results properly
Fixes sorting as well

### CODE REVIEW

**Comment by MikeStall** [comment]
Fixes #1197

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseSuggestionComparer.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            }
+
+            if (_culture == null)
+            {
+#pragma warning disable CA1310 // Specify StringComparison for correctness
+                return x.Text.CompareTo(y.Text);
+#pragma warning restore CA1310 // Specify StringComparison for correctness
+            }
+            
+            return _culture.CompareInfo.GetStringComparer(CompareOptions.IgnoreCase).Compare(x.Text, y.Text);
```
[the remark this answers is not in the document]
Comment (LucGenetier) [reply src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseSuggestionComparer.cs:61]: Carlos told me to be case insensitive
line 57 is current behavior
I agree this isn't super consistent...

**Review by CarlosFigueiraMSFT — APPROVED** [review]
🚢

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**